dpctl/tensor/libtensor/include/kernels/elementwise_functions/equal.hpp

```diff
@@ -151,6 +169,10 @@ template <typename T1, typename T2> struct EqualOutputType
                                         bool>,
         td_ns::
             BinaryTypeMapResultEntry<T1, std::int64_t, T2, std::int64_t, bool>,
+        td_ns::
+            BinaryTypeMapResultEntry<T1, std::uint64_t, T2, std::int64_t, bool>,
+        td_ns::
+            BinaryTypeMapResultEntry<T1, std::int64_t, T2, std::uint64_t, bool>,
```

Comment:
Nice that we can get away with only instantiating int64/uint64 entries in the 4x4 support matrix!